src/Service/SiteSyncService.php

```diff
@@ -140,6 +140,9 @@ public function sync($preview = false)
                     $siteData->setTimezone(isset($site->timeZoneId) ? $site->timeZoneId : null);
                     $siteData->setType($awardee->type);
                     $siteData->setSiteType(isset($site->siteType) ? $site->siteType : null);
+                    if (isset($site->address->state)) {
```

Comment:
If there's the possibility of an address object without a state field, is there also the possibility of a site object without an address? That is, should we check for `(isset($site->address) && isset($site->address->state))`?

Reply:
Yes, I guess it doesn't hurt to check if address field is set.